src/main/java/online/shop/web/filter/AuthorizationFilter.java

```diff
@@ -0,0 +1,80 @@
+package online.shop.web.filter;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import online.shop.lib.Injector;
+import online.shop.model.Role;
+import online.shop.model.User;
+import online.shop.service.UserService;
+
+public class AuthorizationFilter implements Filter {
+    private static final String USER_ID = "userId";
+    private static final Injector injector = Injector.getInstance("online.shop");
+    private final UserService userService = (UserService) injector.getInstance(UserService.class);
+    private Map<String, Set<Role.RoleName>> protectedUrls = new HashMap<>();
+
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
```

Comment:
user orders?

Reply:
There is _/orders/all_(see all user's orders) exist in the previous commit. Also, _/orders_ for admin(see all orders and ability to delete).